src/Routes.tsx

```diff
@@ -1,84 +1,33 @@
 import React, { FunctionComponent } from 'react';
-import { Redirect, Route, Switch } from "react-router-dom";
-import About from "./components/About";
-import Docs from "./components/Docs";
+import { useLocation } from "react-router-dom";
+import { WorkspaceInfo } from './AppContainer';
+import About from './components/About';
+import Docs from './components/Docs';
 import HitherJobMonitor from './components/HitherJobMonitor/HitherJobMonitor';
 import Config from './containers/Config';
 import Home from "./containers/Home";
-import ImportRecordings from "./containers/ImportRecordings";
-import ImportSortings from "./containers/ImportSortings";
-import RecordingView from "./containers/RecordingView";
-import SortingUnitView from './containers/SortingUnitView';
-import SortingView from './containers/SortingView';
-import TimeseriesForRecordingView from "./containers/TimeseriesForRecordingView";
 
-const TestPage = () => {
-    return <div>{`Test page`}</div>;
-}
+const Routes: FunctionComponent<{width: number, height: number, workspaceInfo: WorkspaceInfo}> = ({width, height, workspaceInfo}) => {
+    const location = useLocation()
+    const pathList = location.pathname.split('/')
+    const { page, workspaceName} = (
```

Comment:
This logic for getting the workspace info from the location is duplicated -- will want to consolidate at some point.

Reply:
yes